src/postman/operation.ts

```diff
@@ -20,16 +20,14 @@ import {
 } from './transformers/securityScheme';
 import { transformServer } from './transformers/server';
 import { PostmanCollectionHttpOperationTransformer } from './types';
-import { transformDescriptionDefinition, traverseItemsAndGroups } from './util';
+import { resolveCollection, transformDescriptionDefinition } from './util';
 
 export const transformPostmanCollectionOperations = (document: CollectionDefinition): IHttpOperation[] => {
-  const collection = new Collection(document);
-  const resolvedCollection = new Collection(collection.toObjectResolved({ variables: collection.variables }, []));
```

Comment:
Good call, basically we're doing this always and it makes sense to enclose it in a function.